reference/commands/create.rst

```diff
@@ -59,6 +59,62 @@ The same happens for lockfiles created with ``--lockfile-out`` argument. The loc
   dependencies, you can control their build using the ``--build-test`` argument.
 
 
+Methods execution order
+-----------------------
+
+The ``conan create`` executes :ref:`methods <reference_conanfile_methods>` of a *conanfile.py* in the following order:
+
+#. Export recipe to the cache
+    #. ``init()``
+    #. ``set_name()``
+    #. ``set_version()``
+    #. ``export()``
+    #. ``export_sources()``
+#. Compute dependency graph
+    #. ``ìnit()``
+    #. ``config_options()``
+    #. ``configure()``
+    #. ``requirements()``
+    #. ``build_requirements()``
+#. Compute necessary packages
+    #. ``validate_build()``
+    #. ``validate()``
+    #. ``package_id()``
+    #. ``layout()``
+    #. ``system_requirements()``
+#. Install packages
+    #. ``build_id()``
+    #. ``generate()``
+    #. ``build()``
+    #. ``package()``
+    #. ``package_info()``
+
+After that, if you have a folder named *test_package* in your project or you call the ``conan create`` command with the
+``--test-folder`` flag, the command will call the *conanfile.py* file inside the folder in the following order:
+
+#. Compute dependency graph
+    #. ``ìnit()``
+    #. ``config_options()``
+    #. ``configure()``
+    #. ``requirements()``
+    #. ``build_requirements()``
+#. Compute necessary packages
+    #. ``validate_build()``
+    #. ``validate()``
+    #. ``package_id()``
+    #. ``layout()``
+    #. ``system_requirements()``
+#. Install packages
+    #. ``build_id()``
+    #. ``generate()``
+    #. ``build()``
+    #. ``package()``
+    #. ``package_info()``
```

Comment:
This might not be fully accurate. I am not sure ``package()`` is called for ``test_package/conanfile`` at all.
It would also be missing the call to ``test()`` method.